### PR TITLE
c++ solution of standard graph questions

### PR DESCRIPTION
**Pull request by aryan-basu**

### CODE REVIEW

File: Graphs/Detection of Cycle in directed Graph.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include<bits/stdc++.h>
```
Comment (akashchouhan16) [inline Graphs/Detection of Cycle in directed Graph.cpp:1]: Try to include a comment containing the **Problem statement** with **Sample Test Cases** with the solutions, that have been tested with the code for future PRs. 👨🏻‍💻
Refer : [CONTRIBUTING.md](https://github.com/akashchouhan16/Data_Structures_And_Algorithms/blob/master/CONTRIBUTING.md "View Guidelines")

**Review by akashchouhan16 — APPROVED** [review]
#### Contains a Duplicate Code (`Detect Cycle in Undirected Graph`)
The `PR` needs to be updated and must adhere to [CONTRIBUTING.md](https://github.com/akashchouhan16/Data_Structures_And_Algorithms/blob/master/CONTRIBUTING.md "View Guidelines")

**Review by akashchouhan16 — CHANGES_REQUESTED** [review]
Remove the Duplicate Code to make a valid `PR`.

File: Graphs/Detection of Cycle in directed Graph.cpp (hunk 1 above)
Comment (aryan-basu) [reply Graphs/Detection of Cycle in directed Graph.cpp:1]: sorry i wont have testcases and question for it i will create a fresh pr with platform questions and solution sorry for inconvinenece

**Pull request closed without merging**